### PR TITLE
PIM-4582 : initialize prices collection if not defined

### PR DESCRIPTION
**Pull request by jmleroux**
| Q                    | A
| -------------------- | ---
| Bug fix?             |y
| New feature?         |n
| BC breaks?           |n
| CI currently passes? |y
| Tests pass?          |y
| Scenarios pass?      |y
| Checkstyle issues?*  |n
| PMD issues?**        |n
| Changelog updated?   |n
| Fixed tickets        |PIM-4582
| DB schema updated?   |
| Migration script?    |
| Doc PR               |

### CODE REVIEW

File: features/Context/Page/Product/Creation.php [hunk 1]
```diff
@@ -2,12 +2,15 @@
 
 namespace Context\Page\Product;
 
+use Behat\Mink\Element\NodeElement;
+use Behat\Mink\Exception\ExpectationException;
```
Comment (phaseinducer) [inline features/Context/Page/Product/Creation.php]: Why import this ?

**Comment by phaseinducer** [comment]
GTM

File: features/Context/Page/Product/Creation.php [hunk 2]
```diff
@@ -2,12 +2,15 @@
...
 use Context\Page\Base\Form;
 
 /**
  * Product creation page
  *
  * @author    Gildas Quemener <gildas@akeneo.com>
+ * @author    JM Leroux <jean-marie.leroux@akeneo.com>
```
Comment (sumbobyboys) [inline features/Context/Page/Product/Creation.php]: we bever add authorship after creation. SO you can replace but you will be responsilbe ;)

File: features/product/create_product.feature [hunk 3]
```diff
@@ -23,6 +23,18 @@ Feature: Product creation
     Then I should be on the product "caterpillar_1" edit page
     And I should see "caterpillar_1"
 
+  Scenario: Successfully create a product with family
+    Given I create a new product
+    Then I should see the SKU and Family fields
+    And I fill in the following information in the popin:
+      | SKU    | caterpillar_1 |
+      | Family | Sandals |
```
Comment (sumbobyboys) [inline features/product/create_product.feature]: please align

File: features/product/create_product.feature [hunk 4]
```diff
@@ -23,6 +23,18 @@ Feature: Product creation
...
+    And I press the "Save" button in the popin
+    And I wait 5 seconds
```
Comment (sumbobyboys) [inline features/product/create_product.feature]: noep, we should not do that. Use a spin method if needed

**Pull request merged**